src/odemis/acq/stream/_projection.py

```diff
@@ -31,10 +31,17 @@
 import gc
 import numpy
 
+from odemis.acq.stream import POL_POSITIONS
+from odemis.util.angleres import _CMAP_EELS, _CMAP_MIT
+try:
+    from arpolarimetry import arpolvis
```

Comment:
Probably this can soon be simplified to `import arpolarimetry` (as the only needed function will be in the main module).